### ISSUE TITLE
kit exclaim error

### ISSUE THREAD

**Issue by stephenLYZ** [mikermcneil/kit#3]
hey guy
when i run `kit exclaim hello world` ,it throw an error like:

```
Script encountered an unexpected error.
Details:
----------------------------------------------------------------------
TypeError: Cannot read property 'join' of undefined
TypeError: Cannot read property 'join' of undefined
    at Object.require.fn (/usr/local/lib/node_modules/@mikermcneil/kit/bin/kit-exclaim.js:44:35)
    at afterwards (/usr/local/lib/node_modules/@mikermcneil/kit/node_modules/machine/lib/Machine.prototype.exec.js:750:15)
    at _doCacheLookupMaybe (/usr/local/lib/node_modules/@mikermcneil/kit/node_modules/machine/lib/Machine.prototype.exec.js:376:27)
    at Machine_prototype_exec [as exec] (/usr/local/lib/node_modules/@mikermcneil/kit/node_modules/machine/lib/Machine.prototype.exec.js:417:5)
    at afterMaybeDoingSetup (/usr/local/lib/node_modules/@mikermcneil/kit/node_modules/machine-as-script/index.js:755:62)
    at _doSetupMaybe (/usr/local/lib/node_modules/@mikermcneil/kit/node_modules/machine-as-script/index.js:727:16)
    at Machine.liveMachine.exec (/usr/local/lib/node_modules/@mikermcneil/kit/node_modules/machine-as-script/index.js:744:7)
    at Object.<anonymous> (/usr/local/lib/node_modules/@mikermcneil/kit/bin/kit-exclaim.js:75:4)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
```

**Comment by SlowburnAZ** [comment]
I'm getting the same error here, on Ubuntu

**Comment by SlowburnAZ** [comment]
From this line (line 44 of kit-exclaim.js):
var text = env.commandLineArgs.join(' ');

Seems that env.commandLineArgs is undefined...  weird.

I'm currently running node version 6.5.0, if that helps.

**Comment by mikermcneil** [comment]
@SlowburnAZ @stephenLYao sorry guys!  Had it linked, so I didn't notice the change.  Thanks for the heads up!

**Comment by mikermcneil** [comment]
fix published in 1.2.6
